dotCMS/src/main/java/com/dotcms/rest/api/v1/secret/view/HostView.java

```diff
@@ -0,0 +1,21 @@
+package com.dotcms.rest.api.v1.secret.view;
+
+public class HostView {
+
+    private String hostId;
+    private String hostName;
+
+    public HostView(String hostId, String hostName) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'hostName' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4623512)